src/components/Settings/Export/Export.component.js

```diff
@@ -125,6 +134,32 @@ class Export extends React.Component {
                   </div>
                 </ListItemSecondaryAction>
               </ListItem>
+
+              {isCordova() && (
+                <ListItem>
+                  <ListItemText
+                    primary={'Upload Analytics'}
+                    secondary={'Upload on-disk analytics'}
+                  />
+                  <ListItemSecondaryAction>
+                    <div className="Export__ButtonContainer">
+                      {this.state.uploading && (
+                        <CircularProgress
+                          size={25}
+                          className="Export__ButtonContainer--spinner"
+                          thickness={7}
+                        />
+                      )}
+                      <Button
+                        disabled={this.state.uploading}
+                        onClick={this.onUploadClick}
+                      >
+                        Upload
```

Comment:
This does not allow translation. Use messages from messages file (react-intl).

Reply:
Done